shared/teams/routes.js

```diff
@@ -161,5 +161,10 @@ export const newModalRoutes = {
     getScreen: () => require('./team/really-remove-member/container').default,
     upgraded: true,
   },
+  teamRename: {
+    // TODO add to nav1 routes too
```

Comment:
Can never tell whether a TODO is meant for before or after the merge :)  If after, perhaps file a ticket and put the ticket ID in this comment?  And if before, put up a checkbox in the PR desc?

Reply:
Meant for before merge, done